test/test_cpu/test_export.py

```diff
@@ -25,7 +37,7 @@ class TestAutoRound(unittest.TestCase):
     def setUpClass(self):
         model_name = "facebook/opt-125m"
         self.save_dir = "./saved"
-        self.model = AutoModelForCausalLM.from_pretrained(model_name, dtype="auto", trust_remote_code=True)
+        self.model = AutoModelForCausalLM.from_pretrained(model_name, torch_dtype="auto", trust_remote_code=True)
```

Comment:
how do you sync with main? It seems you fallback some changes in other pr.  Maybe we should use the `update branch` in your pr page to sync